### ISSUE TITLE
Unable to install for OS64COMBINED platform

### ISSUE THREAD

**Issue by doodlewind** [leetal/ios-cmake#87]
**Describe the bug**
The default example-lib fails in install process with the `-DPLATFORM=OS64COMBINED` config.

**To Reproduce**
Using the same operation following the README:

``` bash
cd example-lib
mkdir build && cd build
cmake .. -G Xcode -DCMAKE_TOOLCHAIN_FILE=../../ios.toolchain.cmake -DPLATFORM=OS64COMBINED
cmake --build . --config Release --target install
```

Two `libexample.a` files can be built and found in `build/Release-iphoneos` and `build/Release-iphonesimulator` path, but with the `--target install` config, following error is raised:

``` bash
[iOS combined] Current: /Users/ewind/Downloads/ios-cmake/example/example-lib/../example-app/example-lib/lib/libexample.a
[iOS combined] Corresponding: /Users/ewind/Downloads/ios-cmake/example/example-lib/build/Release-iphonesimulator/libexample.a
fatal error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo: /Users/ewind/Downloads/ios-cmake/example/example-lib/build/Release-iphonesimulator/libexample.a and /Users/ewind/Downloads/ios-cmake/example/example-lib/../example-app/example-lib/lib/libexample.a have the same architectures (arm64) and can't be in the same fat output file
CMake Error at /usr/local/Cellar/cmake/3.15.2/share/cmake/Modules/CMakeIOSInstallCombined.cmake:306 (message):
  Command failed:
  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo;-create;/Users/ewind/Downloads/ios-cmake/example/example-lib/build/Release-iphonesimulator/libexample.a;/Users/ewind/Downloads/ios-cmake/example/example-lib/../example-app/example-lib/lib/libexample.a;-output;/Users/ewind/Downloads/ios-cmake/example/example-lib/../example-app/example-lib/lib/libexample.a
Call Stack (most recent call first):
  cmake_install.cmake:49 (ios_install_combined)


make: *** [install_buildpart_0] Error 1

** BUILD FAILED **


The following build commands failed:
	PhaseScriptExecution CMake\ PostBuild\ Rules build/example-ios.build/Release-iphoneos/install.build/Script-149BD7B82F7D46F2B9BFCFEC.sh
```

**Environment**
* CMake 3.15.2, Xcode Version 12.0.1 (12A7300)
* The CMakeLists.txt is identical to the example: https://github.com/leetal/ios-cmake/blob/master/example/example-lib/CMakeLists.txt

**Comment by leetal** [comment]
Hmmm. I'll investigate this and return to you with my findings. Might be Xcode 12 related.

**Comment by leetal** [comment]
This is sadly a bug in CMake covered here: https://gitlab.kitware.com/cmake/cmake/-/issues/21282 and here https://gitlab.kitware.com/cmake/cmake/-/issues/21281 (closed in favour of 21282)

**Comment by NSShentu** [comment]
@leetal so any workarounds?

**Comment by leetal** [comment]
Nope, nothing has been fixed upstream in CMake to fix this. I'd urge you to try the following potential workaround for the moment: https://github.com/leetal/ios-cmake/issues/88#issuecomment-740656202

**Comment by leetal** [comment]
Fixed in #97. Please see updated README for instructions.
